### DISCUSSION TITLE
Talk on TTS or making machines speak with your voice

### DISCUSSION THREAD

**Discussion by thorstenMueller** [coqui-ai/TTS#515] [General]
Hi.

I've been invited as a speaker by Tensorflow Turkey community to talk about my open german voice dataset contribution and more.
Of course Mycroft and Coqui will be part of my slides/talk.

If you're interested it will be live on youtube on June 2nd, 8pm (german time).

https://twitter.com/ThorstenVoice/status/1398962403862323203

Best
Thorsten

**Comment by erogol** [comment]
Thanks, @thorstenMueller  for mentioning our name and project alongside yours.

**Comment by thorstenMueller** [comment]
The talk is over and Coqui has been an important part of my presentation.
If you're interested see the video here: https://www.youtube.com/watch?v=m-Uwb-Bg144&t=1548s
